doc/dev_plugins.md

```diff
@@ -53,15 +66,19 @@ Some (early stage) plugins are also available as examples:
 
 ## Debugging plugins
 
+`sourmash info -v` will list all detected plugins of any known kind.
```

Comment:
"of any known kind" is a bit confusing to me.

maybe: "any detected plugins that are locally available" ?

Reply:
changed it to:
>all installed plugins

thanks ;)

Reply:
in 7fa809ee